VRDR.Tests/Messaging_Should.cs

```diff
@@ -148,6 +148,13 @@ public void CreateSubmissionFromBundle()
             Assert.Equal(submission.NCHSIdentifier, parsed.NCHSIdentifier);
         }
 
+		[Fact]
+        public void ProtectedSetMessageBundle()
+        {
+            DeathRecordSubmissionMessage submission = new DeathRecordSubmissionMessage();
+            Assert.NotNull(submission);
+		}
```

Comment:
Add a submission.MessageBundle not null assertion here, and reformat the right bracket whitespace.